committee-steering/sig-governance.md

```diff
@@ -0,0 +1,629 @@
+# SIG Governance Template
+
+## Goals
+
+The following document outlines a template governance structure for
+SIGs to follow.  Its goals are to provide the necessary structure
+for SIGs to self organize and self govern.
+
+This document is focused on:
+
+- Outlining organizational responsibilities
+- Outlining organizational roles
+- Outlining processes and tools
+
+Specific attention has been given to:
+
+- The role of technical leadership
+- The role of project leadership
+- The process for making technical decisions
+- The process for ensuring continued test health
+
+## How to use this template
+
+SIGs are diverse in nature, and may have different goals based
+on what areas of the project the SIG touches.  Accordingly SIGs will
+need to adapt this template to their specific needs.  This may mean
+removing or changing roles, responsibilities, and processes.  When
+diverging from this template, please help us understand why in the
+PR description so we can improve the process and iterate on this
+document.
+
+**Note:** There are 2 charter template versions:
+
+- a super-lite TL;DR version that only provides a abstract structure.
+- a detailed version that attempts to concretely define a high
+  degree of structure.
+
+SIGs may either choose to start with the TL;DR charter and add
+structure from the detailed version, or to start with the detailed
+version and remove structure that does not provide direct value.
+
+**Note:** Specific responsibilities towards the project are required
+for some SIGs and should be included in the charter - e.g. ensuring
+test health.
+
+FAQ:
+
+- [Why is it so long?](#why-is-it-so-long)
+- [Why so much structure?](#why-so-much-structure)
+- [Why use mailing lists for tracking roles?](#why-use-mailing-lists-for-tracking-roles)
+- [Why track membership as a special role?](#why-track-membership-as-a-special-role)
+- [How do the results from the survey fit in?](#how-do-the-results-from-the-survey-fit-in)
+
+---
+
+# SIG ~~Foo~~ Governance TL;DR Version
+
+## Roles
+
+- Project Leads
```

Comment:
This role needs another name. I'll try to think of some.

Reply:
What about _SIG Leads_. It's the term often used today. I imagine there has been some discussion and I'm be curious to know the why here.

Thinking in terms of the [principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) I was wondering if a name change is needed as it's a change. If a name change is useful then everyone should be able to articulate why (even if they don't agree with it).

Reply:
I agree with @mattfarina. I don't see a need for a name change.

Reply:
I personally don't have a problem with SIG Lead.

AIUI, the main issue that has been raised by a number of people in the past was that the scope of responsibilities and authority was blurry, but I believe the main reason for that was that we didn't have an official notion of technical leadership or ownership, and that some SIG Leads filled both technical and non-technical roles. Clarifying the technical side might be sufficient, at least within the project.

The other issue raised with the term was that of external perception, triggered primarily by marketing like the following:
https://www.redhat.com/en/about/press-releases/red-hat-acquire-coreos-expanding-its-kubernetes-and-containers-leadership
https://gravitational.com/blog/images/2018/red-hat-sigs.png

Reply:
I don't like the term (SIG Lead), because it conflates the role.  In this document, it's basically a functional manager, and we should strive to ensure that is reflected.

Reply:
I share @timothysc sentiments.  While I don't like the name for the reasons mentioned by @timothysc, I am fine with keeping it if we can't get consensus on something else.